cli/src/commands/utils.ts

```diff
@@ -85,3 +114,40 @@ export function validateAddress(address: string) {
     process.exit(1)
   }
 }
+
+interface WalletConfig {
+  wallet: string
+  password: string
+}
+
+interface Config {
+  wallet_config: WalletConfig
+  node_url: string
+  chunk_size: number
+  secret_key: string
+}
+
+export function parseConfigFile(filePath: string): Config {
+  const fileContent = readFileSync(filePath, 'utf-8')
+  try {
+    const parsedData = JSON.parse(fileContent)
+    return parsedData
+  } catch (error) {
+    throw new Error(`Failed to parse file: ${error}`)
+  }
+}
+
+export function setProgramOptions(
+  commandOptions: OptionValues,
+  configOptions: Config
+): OptionValues {
+  if (!configOptions) return commandOptions
+
+  const wallet = commandOptions.wallet || configOptions.wallet_config.wallet
+  const password =
+    commandOptions.password || configOptions.wallet_config.password
+  const node_url = commandOptions.node_url || configOptions.node_url
+  const chunk_size = commandOptions.chunk_size || configOptions.chunk_size
+
+  return { wallet, password, node_url, chunk_size }
+}
```

Comment:
I think a name like "mergeConfigAndOptions" is clearer ?
```suggestion
export function mergeConfigAndOptions(
  commandOptions: OptionValues,
  configOptions: Config
): OptionValues {
  if (!configOptions) return commandOptions

  const wallet = commandOptions.wallet || configOptions.wallet_config.wallet
  const password =
    commandOptions.password || configOptions.wallet_config.password
  const node_url = commandOptions.node_url || configOptions.node_url
  const chunk_size = commandOptions.chunk_size || configOptions.chunk_size

  return { wallet, password, node_url, chunk_size }
}
```